bootloader/src/pyi_utils.c

```diff
@@ -221,7 +221,7 @@ pyi_unsetenv(const char *variable)
     #if HAVE_UNSETENV
     rc = unsetenv(variable);
     #else /* HAVE_UNSETENV */
-    rc = setenv(variable, "");
+    rc = setenv(variable, "", true);
```

Comment:
Reading ht man-page for `setnv` I have the impression you spotted a long standing bug. Well done!
Unfortunately this change is unrelated to this pull-request. Please provide an extra pull-request. Thanks

Reply:
Ok. I've added that to solve a compile time error, and checking the code the change made some sense.